src/app/key/key.component.html

```diff
@@ -0,0 +1,6 @@
+<div class="key" id="key-{{key}}">
+    <button [ngClass]="largeButton()
```

Comment:
Small thing (especially since this is a small component), but I try to start boolean function names with "is" or "are" to make them sound like yes/no questions. Also, if it's convenient I tend to write functions that don't depend on state besides the function inputs. So this would become `isLargeButton(key)` and `onKeyClick(key)`.